taxonomies/categories.txt

```diff
@@ -22544,40 +22545,47 @@ fr:Crèmes Chantilly, Crème Chantilly, Chantilly
 de:Crème Chantilly
 wikidata:en:Q6384329
 
+<en:Creams
+en:UHT Creams
+fr:Crèmes UHT
+
 <en:Creams
 fr:Crèmes entières, Crème entière
 nl:Volle slagromen
 
-<fr:Crèmes entières
+<en:Creams
+fr:Crèmes liquides, crèmes fraîches liquides, Crème fraîche liquide, Crème liquide
+
+<en:Creams
+fr:Crèmes légères, crèmes fraîches légères, Crème fraîche légère, crème légère
+
+<en:Creams
+fr:Crèmes allégées en matière grasse, Crèmes fraîches allégées en matière grasse, Crème fraîche allégée, Crèmes fraîches allégées, Crème fraîche allegée en matière grasse
+nl:Light zure romen
+
+<fr:Crèmes fraîches
 <en:Sour creams
+fr:Crèmes épaisses, crèmes fraîches épaisses, Crème fraîche épaisse
+
+<en:Creams
+fr:Crèmes fraîches
+
+<fr:Crèmes épaisses
+<fr:Crèmes entières
 fr:Crème Fraîche d'Isigny
```

Comment:
I would not mix up creme and creme fraiche. They are different products with different ingredients,

Reply:
I would add fr:Crème Fraîche d'Isigny straight under en:Sour creams, as it is a fermented cream.